ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/eth/eth_call_blob_without_to.json

```diff
@@ -0,0 +1,22 @@
+{
+  "request": {
+    "id": 4,
+    "jsonrpc": "2.0",
+    "method": "eth_call",
+    "params": [
+      {
+        "from": "a94f5374fce5edbc8e2a8697c15331677e6ebf0b",
+        "data": "0x12a7b914",
+        "blobVersionedHashes" : ["0x0100000051c8833cfbaf272e62da1285b183b0405357f62b052a4894ffcdaa2d"],
+        "maxFeePerBlobGas": "0x3b9aca00"
+      },
+      "latest"
+    ]
+  },
+  "response": {
+    "jsonrpc": "2.0",
+    "id": 4,
+    "error":{"code":-32602,"message":"Invalid transaction type"}
```

Comment:
Note we don't currently get the "transaction must contain a to field" message here - could refactor to make that happen if it's important